src/omnibar/omnibar.ts

```diff
@@ -554,6 +570,26 @@ function buildOmnibarUrl(): string {
   return omnibarUrl;
 }
 
+function updateTitle(): void {
+  let title = '';
+  let titleParts: string[] = [];
+  if (unreadNotificationCount && unreadNotificationCount !== 0) {
+    titleParts.push(`(${unreadNotificationCount})`);
+  }
+
+  if (currentTitleParts && currentTitleParts.length !== 0) {
+    titleParts = titleParts.concat(currentTitleParts);
+  }
+
+  titleParts.push(serviceName);
+
+  title = titleParts.join(' - ');
+
+  if (titleSet) {
```

Comment:
All the other logic in this function should be inside this `if` statement since none of it is necessary when the page's title isn't actually set.